app/editor/components/LinkEditor.tsx

```diff
@@ -196,26 +197,27 @@ const LinkEditor: React.FC<Props> = ({
           value={query}
           placeholder={dictionary.searchOrPasteLink}
           onKeyDown={handleKeyDown}
-          onPaste={handlePaste}
```

Comment:
I believe it's safe to get rid of this? Personally, I don't miss it from a UX point of view, given that a click outside already saves input. It was anyways defunct.

Reply:
Yes, seems fine. Subtly different but not in a terrible way